test/smoke/features/cli/cli_install_dataindex.feature

```diff
@@ -0,0 +1,72 @@
+Feature: CLI: Install Kogito Data Index
+
+  Scenario: CLI install Kogito Data Index
+    Given Kogito Operator is deployed with dependencies
+
+    When CLI install Kogito Data Index with 1 replicas
+    # This line should be removed when https://issues.redhat.com/browse/KOGITO-614 is done
+    And Kafka is installed
+
+    Then Kogito Data Index has 1 pods running within 5 minutes
+    And GraphQL request on service "kogito-data-index" is successful within 2 minutes with path "graphql" and query:
+    """
+    {
+      ProcessInstances{
+        id
+      }
+    }
+    """
+
+#####
+
+### Uncomment those scenarios when custom OLM deployment is in place
+  # Scenario: CLI creates new project with Kogito Data Index (new-project --install-data-index)
+  #   Given Namespace is deleted
```

Comment:
Would it have sense to derive namespace creation to separate step?

Reply:
Working on that :)